src/components/DataDisplay/Chip/Chip.styles.tsx

```diff
@@ -7,22 +7,60 @@ const { colors } = tokens;
 
 interface ChipProps {
   $readonly?: boolean;
+  $variant?: 'active' | 'error' | 'default';
 }
 
+const getColorSchemeBy = (variant: ChipProps['$variant'] = 'default') =>
+  ({
+    error: {
+      color: `${colors.interactive.danger__resting.rgba}`,
+      background: `${colors.ui.background__light.rgba}`,
+      borderColor: `${colors.interactive.danger__resting.rgba}`,
+      hover: {
+        color: `${colors.interactive.danger__resting.rgba}`,
+        background: `${colors.ui.background__light.rgba}`,
+        borderColor: `${colors.interactive.danger__resting.rgba}`,
+      },
+    },
```

Comment:
Wouldn't it be simpler to just have a switch case here? Instead of indexing an object with the key
```typescript
function get(value) {
  switch (value) {
    case 'error':
      return { ... }
}
}
```

Reply:
@mariush2 
Sure we can use a switch statement. 
However imho dispatchment by object key is a bit better due to it being a bit shorter and having the ability to be moved to config files. 😸 

https://hexlet.io/courses/js-polymorphism/lessons/dispatch-data-by-key/theory_unit

What do you think?